mob.go

```diff
@@ -1155,19 +1164,31 @@ func start(configuration Configuration) error {
 		sayInfo("cannot start; clean working tree required")
 		sayUnstagedChangesInfo()
 		sayUntrackedFilesInfo()
-		sayFix("To start, including uncommitted changes, use", configuration.mob("start --include-uncommitted-changes"))
+		fixCommand := configuration.mob("start --include-uncommitted-changes")
+		if configuration.StartCreateRemoteBranch {
+			fixCommand = configuration.mob("start --create --include-uncommitted-changes")
+		}
+		sayFix("To start, including uncommitted changes, use", fixCommand)
 		return errors.New("cannot start; clean working tree required")
 	}
 
 	git("fetch", configuration.RemoteName, "--prune")
 	currentBaseBranch, currentWipBranch := determineBranches(gitCurrentBranch(), gitBranches(), configuration)
 
-	if !currentBaseBranch.hasRemoteBranch(configuration) {
+	if !currentBaseBranch.hasRemoteBranch(configuration) && !configuration.StartCreateRemoteBranch {
 		sayError("Remote branch " + currentBaseBranch.remote(configuration).String() + " is missing")
-		sayFix("To set the upstream branch, use", "git push "+configuration.RemoteName+" "+currentBaseBranch.String()+" --set-upstream")
+		sayFix("To start and set the upstream use", "mob start --create")
```

Comment:
We should tell them this will create a remote branch

Reply:
isn't it enough to say that it sets the upstream? Isn't that the same than creating a remote branch?

Reply:
The way I understand it is that you can create a local branch "a", and you can then create a remote branch "origin/b", and they are not connected in any way. Even the names are different, "a" vs "b". Then you can choose to tell your local git to connect your local branch "a" to the remote branch "origin/b", by saying "--set-upstream".
So I'm thinking set-upstream and create a remote branch are two different things, and set-upstream is just doing the connection between the two in your local git.
But maybe you are right, and we don't yet create the remote branch. This will happen only later as we push?

Reply:
okay I see your point. I knew that this is possible but i never saw a usecase for that, because of this for me it felt similar. But then I will rename it and call it "create a remote branch". No the remote branch is directly created and pushed so i think the naming above is correct.